src/popup/locales/en.json

```
@@ -738,15 +738,15 @@
       "seeTransactionExplorer": "See transaction in the explorer.",
       "scan": "Scan QR",
       "cancel": "Cancel",
-      "review": "Review",
+      "review": "Review and send",
       "reviewtx": "Review Transaction",
       "checkalert": "Please check the information carefully.",
       "sender": "Sending address",
       "recipient": "Receiving address",
       "amount": "Amount",
-      "editTxDetails": "Edit Transaction Details",
+      "editTxDetails": "Edit transaction",
       "tx-success": "Transaction Completed",
-      "successalert": "You have successfully sent ",
+      "successalert": "has been sent to",
```

```suggestion
      "successAlert": "has been sent to",
```
For now we don't have a strong opinion on that, but it should be either camelCase or kebab-case, as every other variables.